hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/txns/submission/BasicSubmissionFlowTest.java

```diff
@@ -101,7 +101,7 @@ void rejectsInvalidAccessor() {
 
     @Test
     void followsHappyPathToOk() {
-        setupStakedNode();
+        setupNonZeroWeightNode();
```

Comment:
Weight is derived from stake. We can set up a node's stake and weight is calculated. So I think `setupNonZeroStakeNode`

Reply:
Fixed